experimental/python/wheel.bzl

```diff
@@ -209,6 +212,11 @@ _entrypoint_attrs = {
     "console_scripts": attr.string_dict(
```

Comment:
The .bzl rule can just merge console_scripts into entry_points/plugins (i.e. treat entries in "console_scripts" as if they were entries in "entry_points" in the "console_script" group). I guess the question is whether console_scripts should be marked as deprecated or do we want to live with this merging forever. I'm fine with marking as deprecated.

Reply:
Implemented.

I documented `console_scripts` as deprecated based on how other things in this project are. I didn't see a specific Starlark flag to make an attr as deprecated.

Reply:
I think the best we can do is print("console_scripts is deprecated, please use entry_points instead") in the skylark rule when it sees non-empty console_scripts.

Reply:
Done.

Reply:
Hmm, this `print` statement fails `buildifier`. Is there an alternative?